### ISSUE TITLE
Set focus properly when widget opened

### ISSUE THREAD

**Issue by ctSkennerton** [ctSkennerton/pygmy#7]
Ideally when the search box gets shown the keyboard focus should be immediately set to the line edit widget so that the user doesn't need to click on it to start typing. Also pressing enter in the line edit widget should be equivalent to pressing the find button. Lastly, the focus should be set such that pressing tab in the search box should set the focus to the find button
